src/battle_script_commands.c

```diff
@@ -9640,6 +9669,25 @@ static void Cmd_various(void)
         }
         return;
     }
+    case VARIOUS_TIDY_UP:
```

Comment:
I think I might have seen somebody comment about this on #expansion-dev, but we've moved away from defining new `various`es and now define `callnative`s because `various`es are harder to read, and can be a merge conflict magnet for downstream users.